cloudbuild.yaml

```diff
@@ -56,8 +56,8 @@ steps:
   - 'CLOUD_BUILD_BRANCH_NAME=${_CLOUD_BUILD_BRANCH_NAME}'
   - 'DPU_IMAGE_NAME=${_DPU_IMAGE_NAME}'
   - 'CLOUD_BUILD_PAT_TRIGGER_ID=${_CLOUD_BUILD_PAT_TRIGGER_ID}'
-  - 'SECRET_KEY=${_SECRET_KEY}'
-  - 'NEXT_PUBLIC_SECRET_KEY=${_SECRET_KEY}'
+  - 'GITHUB_PAT_ENCRYPTION_SECRET_KEY=${_GITHUB_PAT_ENCRYPTION_SECRET_KEY}'
```

Comment:
This is not required. You can use the publicly available environment variables everywhere. 

Whenever in doubt (or whenever you see redundancy), refer the documentation, or first test with the non-redundant variant.